test/take.test.js

```diff
@@ -0,0 +1,13 @@
+const { take } = require("../lib");
+
+test("Should return empty array if count is zero", () => {
+  expect(take([1,2,3],0)).toBe([]);
```

Comment:
These functions should use `toEqual` and not `toBe`, as `toBe` checks for same identity.

Reply:
done !